README.md

```diff
@@ -35,7 +35,14 @@ Features
 - Support discard cached connections to foreign servers by using function `sqlite_fdw_disconnect()`, `sqlite_fdw_disconnect_all()`.
 - Support Bulk `INSERT` by using `batch_size` option
 - Support `INSERT`/`UPDATE` with generated column
-- Support `ON CONFLICT DO NOTHING`.
+- Support `ON CONFLICT DO NOTHING`
+- Support mixed SQLite [data affinity](https://www.sqlite.org/datatype3.html) input and filtering (`SELECT`/`WHERE` usage) for such dataypes as
+	- `timestamp`: `text` and `int`,
+	- `uuid`: `text`(32..39) and `blob`(16),
+ 	- `bool`: `text`(1..5) and `int`.
+- Support mixed SQLite [data affinity](https://www.sqlite.org/datatype3.html) output (`INSERT`/`UPDATE`) for such dataypes as
+	- `timestamp`: `text`(default) or `int`,
+ 	- `uuid`: `text`(36) or `blob`(16)(default).
```

Comment:
How about insert `bool`?

Reply:
Thank for your explaining. I understood.    
Please help me to resolve this thread.